vcpkg.json

```diff
@@ -40,14 +40,16 @@
       "name": "opencv4",
       "default-features": false,
       "features": [
-        "ffmpeg",
         "jpeg"
       ]
     },
     {
       "name": "tomlplusplus",
       "version>=": "3.1.0"
     },
+    {
+      "name": "rocal"
```

Comment:
Now that you have it working, let's add it cleanly as a feature. Add a `rocal` entry under `features` in this file, similar to vitis and migraphx, and move this dependency there. Add it to the main CMakeLists as for vitis [here](https://github.com/Xilinx/inference-server/blob/main/CMakeLists.txt#L88-L90). Add a rocal flag into the [dockerfile](https://github.com/Xilinx/inference-server/blob/main/docker/template.dockerfile#L302) and update [install_vcpkg.sh](https://github.com/Xilinx/inference-server/blob/main/docker/install_vcpkg.sh) as well. IIRC, this is just an optimization to build rocal dependencies at container build time rather than at server build time.

Reply:
If I move it to features, should I copy over all the dependencies from [here](https://github.com/hansely/inference-server/blob/H/rocal/external/vcpkg/ports/rocal/vcpkg.json#L9) to under features? 
Or do I remove them and move it to under features?

Reply:
No. The port definition you've added for rocal is independent of the inference server. At the inference server vcpkg.json level, the dependency is specifically on rocal as a feature. If rocal is present, then whatever your dependencies are, as specified in rocal's vcpkg.json, get brought in as well.